Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -1069,13 +2216,13 @@ def get_model_description(client: Client, model, id):
     return fileResult(F"{model}_{id}.html", description.encode(encoding='UTF-8'))
 
 
-def get_iocs_by_model(client: Client, model, id, limit="20"):
+def get_iocs_by_model(client: Client, model, id, limit="20", page=None, page_size=None):
     """
         Returns list of indicators associated with specific Threat Model by model id.
     """
-    params = dict(limit=limit)
+    params = return_params_of_pagination_or_limit(arg_to_number(page), arg_to_number(page_size), arg_to_number(limit))
     model_type = model.title()
-    response = client.http_request("GET", F"v1/{model}/{id}/intelligence/", params=params, resp_type='response')
+    response = client.http_request("GET", F"v1/{model.replace(' ', '')}/{id}/intelligence/", params=params, resp_type='response')
```

Comment:
Why this addition?

Reply:
I added a model to the yml named "attack pattern" and the API endpoint is without space so I add this replace

Reply:
Not sure I understand. What the API is expected to get? 
If it's an "attack pattern" without the space, then check it before it and pass as it should.
No need to change the suffix like this